_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+---
+title: ApiaryUI - Interactive Documentation v4
+excerpt: Introduction version 4 of Apiary Interface Documentation
+layout: post
+date: 2018-12-24 12:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
+
+ApiaryUI has already been deployed to [apiary.io](https://apiary.io/) on 20th of December 2018. 
+ApiaryUI is currently in what we call EAP (Early Access Program).
+Although it's already GA (Generally Available), it’s not yet fully production ready.
+We want to share it with the world now and get the invaluable feedback from you - Apiary users.
+
+ApiaryUI is implemented in a way, that allows complete separation from [apiary.io](https://apiary.io/)
+and possibly forming a new separate product in the future. Some features from v3 are still missing
+in ApiaryUI. Some requested features not present in v3 were implemented to ApiaryUI.
+We’ve managed to fix tons of v3 bugs inside the new ApiaryUI. 
+
+As the architecture of ApiaryUI is very simple and solid, we are able to add new features
+and fix bugs in very rapid pace. Our extensive suite of unit, integration and e2e tests
+are helping with that. This can finally release further evolution of [API Blueprint](https://apiblueprint.org/) language.
+
+We are currently in a process of creating the deprecation strategies for  
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/)
+and [Embed](https://help.apiary.io/tools/embed/). When we have those strategies fully formed
+we'll them announce on [Apiary Blog](https://blog.apiary.io/).
+
+
+### What's new in ApiaryUI
+
+ApiaryUI went through major product decisions. The concept of having three column layout stayed,
+but we reorganized the information displayed on each of them. Human Column is now designed
+to contain all the information needed for API Viewer to fully understand what the API is all about.
+Machine Columns sole responsibility is to display machine processing related information
+about one Http Transaction - Response/Request pair.
+
+Although ApiaryUI exclusively uses [API Elements](https://apielements.org/) as data format, it can also consume
+Api Description Document (ADD) formats like [API Blueprint](https://apiblueprint.org/) or [OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) ([OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) coming soon)
+and it parses the ADDs in Web Worker inside the browser. This means that ApiaryUI is completely
+backend-less and does not need the server as it's backend.
```

Comment:
Redundant: either "does not need server" (preferred) or "is completely backend-less".

Reply:
Already removed in previous comments. `is completely
backend-less` used.